src/Drupal/WebformTrait.php

```
@@ -34,6 +75,56 @@ public function webformDelete(string $title): void {
+   *   The title (or partial title) of the template to clone.
+   *
+   * @code
+   *   Given a webform "My contact form" from template "Contact"
+   * @endcode
+   */
+  #[Given('a webform :title from template :template')]
+  public function webformCloneTemplate(string $title, string $template): void {
+    $templates = $this->webformTemplates($template);
+
+    if (empty($templates)) {
+      throw new \RuntimeException(sprintf('No webform template matching "%s" was found.', $template));
```

PHPMD (2.15.0)

[error] Missing class import via use statement (line '98', column '17'). (undefined)

(MissingImport)